release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -361,6 +361,25 @@ With this release, you can now provision an {product-title} cluster with Tang-en
 
 For more information, see xref:../installing/install_config/installing-customizing.adoc#installation-special-config-encryption-threshold_installing-customizing[Configuring an encryption threshold] and xref:../installing/install_config/installing-customizing.adoc#installation-special-config-storage-procedure_installing-customizing[Configuring disk encryption and mirroring].
 
+[id="ocp-4-14-new-cert-process"]
+==== Certificates are now handled by the Machine Config Daemon
+
+In previous {product-title} versions, the MCO read and handled certificates directly from machine configuration files. This led to rotation issues and created unwanted situations, like when certificates were stuck behind a paused machine config pool.
+
+With this release, certificates are no longer templated from bootstrap into machine configuration files. Instead, they are put directly into the Ignition object, written onto disk using the controller config, and handled by the Machine Config Daemon (MCD) during regular cluster operation. The certs are then visible using the Controller Config.
+
+The MCD handles the following certificates:
+
+* `/etc/kubernetes/kubelet-ca.crt`
+* `/etc/kubernetes/static-pod-resources/configmaps/cloud-config/ca-bundle.pem`
+* `/etc/pki/ca-trust/source/anchors/openshift-config-user-ca-bundle.crt`
+
+This means that certificates are not bound by the machine config pool status and are more timely in their rotation.
+
+The above listed CAs stored in machine configuration files have been removed, and the templated files found during cluster installation no longer exist.
```

Comment:
```suggestion
The previously listed CAs stored in machine configuration files are removed, and the templated files found during cluster installation no longer exist.
```

Reply:
IBMSG adivises caution when using above and suggests using previously or preceding instead. 
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#a

Also, not sure if this makes sense here, but I was thinking active voice is better.

Reply:
totally makes sense 🙂